05/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>mobile desktop website</title>
+		<link rel="stylesheet" href="./style.css" />
+	</head>
+	<body>
+		<header class="header">
+			<nav class="header__nav wrapper">
+				<div class="header__nav--logo"></div>
```

Comment:
Tutaj zdecydowanie nie powinny występować Modyfikatory, a raczej Elementy z BEM. Modyfikator to specjalny przypadek wyglądu dla Elementu - dlatego on sam nie występuje. Przykład: Mamy elementy oferty i jeden z nich jest oznaczony jako wybrany. Wtedy ten wybrany może mieć Modyfikator, który styluje ten "wybór".